### ISSUE TITLE
Make minor copy changes to duplicate patient modal

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#2644]
# Description

SimpleReport performs a duplicate patient check on both the self-registration workflow as well as the "Add Person" admin form. In both cases, the modal displayed when an existing patient is found contains the same text. This may be confusing in the case where a user at a testing site registers a patient since the modal states that "you" (i.e. the test site user) already has an account (see pic). This is misleading because it is the _patient_ that has an account, not the test site user.

![image (2).png](https://images.zenhubusercontent.com/604fb43ab2c0ce6f2a47191e/b94e4a92-a887-460c-9d70-3b43d9828d4a)

# Acceptance Criteria
* The duplicate patient modal properly indicates the entity attempting to be registered

**Comment by Rebecca-LM** [comment]
@nathancrtr Do we need to do anything with this message as well that [Adam mentioned](https://skylight-hq.slack.com/archives/C01SY3DRMNY/p1632278067072900?thread_ts=1632276579.072700&cid=C01SY3DRMNY) in his thread? Or will it only be shown on self-register? If nothing else do need to change "testing site" to "testing facility" since that's the official name we're supposed to use for it.

![image (1)](https://user-images.githubusercontent.com/72945498/134419186-b1926360-9e24-4582-a738-b889ab0c7454.png)

**Comment by nathancrtr** [comment]
@Rebecca-LM correct, the version of the modal that appears in that screenshot _only_ appears during self-registration. I think the language there is OK, outside of the "site" -> "facility" change you mentioned.

**Comment by Rebecca-LM** [comment]
@nathancrtr ok here are my suggested changes for both messages mentioned:
- "testing site" to "testing facility" and
- do not" to "don't"

For the original message on this ticket we can just change the header to "This person is already registered at xx."

For the one I posted (for consistency) change header to "You already have a profile at xx" to "You're already registered at xx." The reason for this is I think it's more clear to say they're registered vs. mentioning a profile, which is a bit more vague.

**Comment by nathancrtr** [comment]
@Rebecca-LM for the non-self-registration view, we'll need to change the pronouns from "you" to "this person" in the body of the modal, rather than just the header:

>Our records show someone has registered with the same name, date of birth, and ZIP code. **Please check in with your testing facility staff. You don't need to register again.**

How should we treat the bolded text - maybe just delete it entirely in this case?
